### PR TITLE
Adds Dockerized Logspout & ELK Stack

### LINKED ISSUE #101: Send Docker container's logs to Logstash
The logback-logstash integration is JVM specific (i.e. won't work for other tech) and has been somewhat flaky at times. It also uses TCP which might affect performance of the running application. Explore ways to send logs from running Docker container to Logstash. At the same time, review codebase to standardize and increase logging

### PR DESCRIPTION
**Pull request by hkeeler**
Closes #101.

Here's the first pass at a setting up a distributed logging system for grasshopper.  It uses the _Official_ Official [Elasticsearch](https://registry.hub.docker.com/_/elasticsearch/) (existing) and [Logstash](https://registry.hub.docker.com/_/logstash/) images, semi-Official [gliderlabs/logspout](https://registry.hub.docker.com/u/gliderlabs/logspout/) image (gliderlabs wrote logspout), and [marcbachmann/kibana4](https://registry.hub.docker.com/u/marcbachmann/kibana4/) image (most feature-rich and :star:'d  Kibana image).

To fire it up, you just need to do a:

```
docker-compose up
```

To view logstash output:

```
curl http://{{boot2docker-ip}}:8000/logs/
```

To check out Kibana, browse to:

```
http://{{boot2docker-ip}}:5601/
```

### CODE REVIEW

**Comment by hkeeler** [comment]
I think this is _good enough_ for a first pass at a Docker ELK stack.  It currently supports parsing of Nginx and Gunicorn access logs, Elasticsearch, and Grasshopper's Logback-configured logs.  It does **not** parse multi-line log statements yet, so pretty-formatted JSON, stack traces, etc. are not currently captured as a single log entry.  I can tackle this whenever that becomes a priority.

File: addresspoints/src/main/resources/logback.xml [hunk 1]
```diff
@@ -1,23 +1,14 @@
...
-        <remoteHost>${LOGSTASH_HOST:-localhost}</remoteHost>
-        <port>${LOGSTASH_PORT:-9001}</port>
-
-        <!-- encoder is required -->
-        <encoder class="net.logstash.logback.encoder.LogstashEncoder" />
-    </appender>
-
-    <root level="INFO">
-        <appender-ref ref="logstash" />
+    <root level="DEBUG">
```
Comment (jmarin) [inline addresspoints/src/main/resources/logback.xml]: @hkeeler I think level="INFO" should be the default, DEBUG produces to much output and will probably slow down running service in production

**Comment by jmarin** [comment]
@hkeeler Done reviewing this, only a comment on log level. Other than that, looks good to me.

**Comment by hkeeler** [comment]
Cool.  I will roll back the log level.  I _think_ I set that to `DEBUG` while trying to the Akka HTTP [`DebugDirectives.logRequestResult`](http://doc.akka.io/docs/akka-stream-and-http-experimental/1.0-RC3/scala/http/routing-dsl/directives/debugging-directives/logRequestResult.html#logrequestresult) working...which I never could.  I'll create a new issue for that.

**Comment by hkeeler** [comment]
Added #113 related to `logRequestResult`.

**Comment by hkeeler** [comment]
...and log level rolled back as well.  Should be good-to-go.

**Pull request merged**